src/components/SingleBook.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const SingleBook = () => (
+  <div>
+    <span>Book Categories</span>
+    <h2>book title</h2>
+    <ul>
+      <li>Comments</li>
+      <li>Remove</li>
+      <li>Edit</li>
+    </ul>
+  </div>
+);
+
+export default SingleBook;
```

Comment:
- [ ]  You might want to consider adding a remove button (no functionality yet). 😃